docs/Contributing/Issues/issues.md

```diff
@@ -0,0 +1,67 @@
+## Using the issue tracker
+
+The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs),
+features requests and for submitting pull
+requests, but please respect the following restrictions:
+
+* Please **do not** use the issue tracker for personal support requests. [The discussions tab](https://github.com/RI-SE/ATOS/discussions) on the repo can be used for this instead. 
+
+* Please **do not** derail or troll issues. Keep the discussion on topic and
+  respect the opinions of others.
+
+
+<a name="bugs"></a>
+## Bug reports
+
+A bug is a _demonstrable problem_ that is caused by the code in the repository.
+Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `main` or development branch in the repository.
+
+3. **Isolate the problem** &mdash;
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What OS are you using, which
+browser is used for the gui?What would you expect to be the outcome? All these
```

Comment:
The questions could be in list form

Reply:
Most of these steps are included under the other headings on this page so this might get excluded.